generators/client/templates/angular/src/main/webapp/app/admin/user-management/_user.service.ts

```diff
@@ -20,11 +20,16 @@ export class UserService {
         return this.http.get(<% if (authenticationType === 'uaa') { %>`<%= uaaBaseName.toLowerCase() %>/api/users/${login}`<%} else { %>`api/users/${login}`<% } %>).map((res: Response) => res.json());
     }
 
-    query(req: any): Observable<Response> {
+    query(req?: any): Observable<Response> {
         let params: URLSearchParams = new URLSearchParams();
-        params.set('page', req.page);
-        params.set('size', req.size);
-        params.set('sort', req.sort);
+        if(req) {
+            params.set('page', req.page);
+            params.set('size', req.size);
+            if(req.sort){
+                params.paramsMap.set('sort', req.sort);
```

Comment:
I had problems when sorting with other than the id column. In those cases there are 2 sort params sent, the user sorted column and id column. That case did not work with params.set since the sort parameter was an array with 2 elements.
https://github.com/jhipster/generator-jhipster/blob/entity-ng2/generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.component.ts#L45